earl/src/main/java/com/einmalfel/earl/RdfItem.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * This file is part of Fineswap Android App.
+ * Copyright (C) Fineswap AS. All rights reserved.
+ *
+ * THIS IS PROPRIETARY SOFTWARE AND IS OWNED BY FINESWAP AS.
+ * DO NOT DISTRIBUTE! DO NOT MODIFY! DO NOT COPY!
+ */
+
+package com.einmalfel.earl;
+
```

Comment:
according to google's java style guide, which I'm trying to follow in Earl, there should be single blank line